inference-engine/tests/functional/inference_engine/ngraph_reader/gather_nd_tests.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
2018-2020 -> 2020

Reply:
I moved some tests for `GatherND` from gather_tests.cpp so I left copyright year since 2018.